packages/jellyfish-buffer/src/composer.ts

```diff
@@ -532,6 +532,28 @@ export abstract class ComposableBuffer<T> implements BufferComposer {
     }
   }
 
+  /**
+   * VarUInt1 helper method, 1 - 9 bytes
+   *
+   * @param getter to read from to buffer
+   * @param setter to set to from buffer
+   * @param asC map single object into ComposableBuffer Object
+   */
+  static varUInt1<T> (
```

Comment:
refer to single
https://github.com/DeFiCh/jellyfish/blob/67f34f27b6c2d0e7ba6d25a86e80d30aace4f09e/packages/jellyfish-buffer/src/composer.ts#L164-L184

Reply:
Ya, Suranga is going to fix this.

Reply:
I have just changed to `single`